src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -287,7 +272,7 @@ private function getMinifyParameter()
     /**
      * @throws ViewRenderException
      */
-    private function createDirectoryIfNeeded(string $dir): bool
+    private function createDirectoryIfNeeded(string $dir): void
```

Comment:
Could you also add the $dir argument to the docblock (if there is a docblock, it should contain all "standard" information)?

Reply:
ok